### ISSUE TITLE
Jekyll Migration ?s

### ISSUE THREAD

**Issue by raygunray** [raygunray/reaganray.com#6]
Trent- Any reason why I'd lose my webfonts? It's just showing the default san-serif for me. Do I need to do anything @ Typekit?

**Comment by raygunray** [comment]
Ok - working on the site tonight and just gonna use this issue for some general questions. No rush on any of it. 

**Site-header**
* On sub pages I'd like the H1/Name + nav links to be black. I'd like to bring the hover down to .08.
* The work nav link should actually jump down to the client work section. Is it possible to do that from a subpage? 

**Images**
* Still have that weird issue where the right part of the shadow is cut off when the image is on the right-hand edge of the container.

**Comment by raygunray** [comment]
Update:

* Got typekit working. It needed my 127.0.0.1 in the kit settings after switching to Jekyll.
* Go the H1/nav links to be black. But for some reason when I change the hover to .08 on subs.scss, it changes it everywhere.

**Comment by TrentWalton** [comment]
> The work nav link should actually jump down to the client work section. Is it possible to do that from a subpage?

Tiny change to add a / to the link

> Still have that weird issue where the right part of the shadow is cut off when the image is on the right-hand edge of the container.

An `overflow: hidden;` issue. I commented it out for `.patchwork` so let’s see if it breaks anything. I kept it on `.container`, but added 5px of padding on the left and right (and increased the width of the container by 2%). Seems legit, I think.

**Comment by raygunray** [comment]
OK, I think I'm p close here. Some things I've done-

* Increased the font-size sitewide. Took a lot of finessing, but I think I got it. Check how it squishes for me. 
* Added all the project/info content.
* Styled the blog. I decided to just stick with the post title index page. I'm digging it. Only think I need help with is paginating that page. Maybe ~10 posts per page?
* Click around and see if there's anything else I've missed.

Thanks dude! No rush, of course. Just wanted to get this all down.
